views/tag_view.py

```diff
@@ -0,0 +1,49 @@
+import sqlite3
+import json
+
+def list_tags():
+    # Open a connection to the database
+    with sqlite3.connect("./db.sqlite3") as conn:
+        conn.row_factory = sqlite3.Row
+        db_cursor = conn.cursor()
+
+        # Write the SQL query to get the information you want
+        db_cursor.execute("""
+        SELECT
+            t.id,
+            t.label
+        FROM Tags t
+        """)
+        query_results = db_cursor.fetchall()
+
+        # Initialize an empty list and then add each dictionary to it
+        tags=[]
+        for row in query_results:
+            tags.append(dict(row))
+
+        # Serialize Python list to JSON encoded string
+        serialized_tags = json.dumps(tags)
+
+    return serialized_tags
+
+def retrieve_tag(pk):
+    # Open a connection to the database
+    with sqlite3.connect("./db.sqlite3") as conn:
+        conn.row_factory = sqlite3.Row
+        db_cursor = conn.cursor()
+
+        # Write the SQL query to get the information you want
+        db_cursor.execute("""
+        SELECT
+            t.id,
+            t.label,
+        FROM Tag t
```

Comment:
Looks like this needs to be selecting from 'Tags' instead of 'Tag'  when we query one specific tag